Instagram/View Controllers/LoginViewController.m

```diff
@@ -0,0 +1,116 @@
+//
+//  LoginViewController.m
+//  Instagram
+//
+//  Created by Mai Ngo on 6/27/22.
+//
+
+#import "LoginViewController.h"
+#import "Parse/Parse.h"
+#import "Foundation/Foundation.h"
+
+@interface LoginViewController ()
+@property (weak, nonatomic) IBOutlet UITextField *usernameField;
+@property (weak, nonatomic) IBOutlet UITextField *passwordField;
+
+@end
+
+NSString *const loginSegue = @"loginSegue";
+
+@implementation LoginViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    // Do any additional setup after loading the view.
+}
+- (IBAction)onTapRegister:(id)sender {
+    [self registerUser];
+}
+- (IBAction)onTapLogin:(id)sender {
+    dispatch_semaphore_t sema = dispatch_semaphore_create(0);
+
+    NSMutableURLRequest *const request = [NSMutableURLRequest requestWithURL:[NSURL URLWithString:@"https://api.upcitemdb.com/prod/trial/lookup?upc=079298000085"]
```

Comment:
This looks like you were testing code for your other project--might want to revert it back.